.travis.yml

```diff
@@ -32,7 +32,7 @@ env:
     # Note: env entry here must exactly match the value in the exclude: table below that contains env:, otherwise it will not find a match.
     - CONFIG_OPTS="" DESTDIR="_install"
     - CONFIG_OPTS="no-asm -Werror --debug no-afalgeng no-shared enable-rc5 enable-md2 -fsanitize=address" LSAN_OPTIONS="report_objects=1"
-    - CONFIG_OPTS="no-asm no-makedepend enable-buildtest-c++ --strict-warnings -D_DEFAULT_SOURCE" BUILDONLY="yes" CHECKDOCS="yes" CPPFLAGS="-ansi"
+    - CONFIG_OPTS="no-asm no-makedepend enable-buildtest-c++ --strict-warnings -D_DEFAULT_SOURCE" BUILDONLY="yes" CHECKDOCS="yes" MARKDOWNLINT="yes" CPPFLAGS="-ansi"
```

Comment:
Instead of adding MARKDOWNLINT here, I would add it on line 138 - that is the "disable all that's possible" build which should be already quite fast so running something else in there should not be a problem

Reply:
thanks, good idea.